### PR TITLE
Add Implement the return Rental functionality and Rental date validation

### PR DESCRIPTION
**Pull request by nklimovych**

### CODE REVIEW

**Review by azzilian — APPROVED** [review]
Gj, I will check my code so removal of userId from repsonseDto wont affect telegram

**Review by azzilian — DISMISSED** [review]

File: src/main/java/online/carsharing/dto/response/rental/RentalResponseDto.java [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
 @Data
 public class RentalResponseDto {
     private Long id;
     private LocalDate rentalDate;
     private LocalDate returnDate;
     private LocalDate actualReturnDate;
-    private Long carId;
-    private Long userId;
+    private Car car;
```
Comment (Recdt) [inline src/main/java/online/carsharing/dto/response/rental/RentalResponseDto.java]: Add car responce dto against car

**Review by Recdt — DISMISSED** [review]
GJ! 1 small comment

File: src/main/java/online/carsharing/dto/response/rental/RentalResponseDto.java (hunk 1 above)
Comment (nklimovych) [reply src/main/java/online/carsharing/dto/response/rental/RentalResponseDto.java]: Thanks, done!)

**Review by azzilian — DISMISSED** [review]

**Review by azzilian — APPROVED** [review]

**Review by Recdt — APPROVED** [review]

**Review by HAIOVYI — APPROVED** [review]
GJ!

**Pull request merged**